visual-e2e/examples/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
```

Comment:
Can you update this repo to use V7 together with async mode, see my comments on one of the test files